tests/Drupal/Commenting/TodoCommentUnitTest.inc

```diff
@@ -7,9 +7,9 @@
  * These are valid examples.
  *
  * @todo Valid.
- * @todo valid with lower-case first letter
- * @todo $can start with a $
- * @todo \also with backslash
```

Comment:
Same here, backslash should be valid. "@todo \Drupal should not be used here, implement dependency injection later".